modules/core/src/test/java/org/apache/ignite/testframework/GridTestUtils.java

```diff
@@ -2587,6 +2589,20 @@ public static long sleep_and_can_fail() {
         }
     }
 
+    /**
+     * @param lowerBound Lower bound.
+     * @param upperBound Upper bound.
+     */
+    public static <T extends Comparable<? super T>> Matcher<T> inRange(T lowerBound, T upperBound) {
+        return new CustomMatcher<T>("should be in range [" + lowerBound + ", " + upperBound + "]") {
+            @SuppressWarnings({"unchecked", "rawtypes"})
+            @Override public boolean matches(Object item) {
+                return lowerBound != null && upperBound != null && item instanceof Comparable
```

Comment:
Lead to unexpected behaviour when called with nulls, for example assertThat(5, inRange(null, 10) should return True (for me)... Can we check arguments here or annotate its with \@NotNull  ?

Reply:
Added not null check. BTW we don't use `@NotNull` annotation in the project